ui/src/groups/GroupSidebar/MobileGroupSidebar.tsx

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
```

Comment:
A little confused about the changes for this nav. we removed the ability to see group filtered notifications and group actions, and still don't have a way to leave the group. 

Seems like notifications should be top level and invite should be in the ellipsis menu with the other things I mentioned?